tests/lib/rules/no-children-prop.js

```diff
@@ -15,6 +15,7 @@ var RuleTester = require('eslint').RuleTester;
 var parserOptions = {
   ecmaVersion: 6,
   ecmaFeatures: {
+    experimentalObjectRestSpread: true,
```

Comment:
I'm not sure why - this is the PR that adds `experimentalObjectRestSpread`. but i don't think this needs to block if other maintainers think it's fine.